aep/general/0004/aep.md.j2

```diff
@@ -16,14 +16,40 @@ tools such as Kubernetes or GraphQL interact with APIs from multiple providers.
 APIs **must** define a resource type for each resource in the API, according to
 the following pattern: `{API Name}/{Type Name}`. The type name:
 
+### API Name
+
+An API name is a string that **must** uniquely identify the API. In practice,
+it is difficult to prevent the re-use of the name across different APIs, and
+therefore the API should use something, such as the domain name of the primary
```

Comment:
```suggestion
it is difficult to prevent the re-use of the name across different APIs. 
Therefore the API should use something, such as the domain name of the primary
```